doc/_includes/navigation.html

```diff
@@ -0,0 +1,19 @@
+<nav class="doc-nav doc-nav-internal" role="navigation" aria-label="{{ include.title }}">
+    <h2 class="doc-heading doc-nav-title hd-6">{{ site.data.edx-ui-toolkit.name }}</h2>
+    <ul class="list-unstyled list-patterns">
+        <li class="item no-indent">
+            <a class="doc-link" href="{{ site.baseurl }}">
+            Introduction
+            </a>
+            <a class="doc-link" rel="external" href="{{ site.data.edx-ui-toolkit.url_github }}">
+            GitHub Repository
```

Comment:
I found this link to be jarring. It's in the left navigation tree so I expected to open another page in the UI toolkit doc. Navigating to the GitHub site was unexpected.

I think a link to the GitHub repository would be good content on the intro page ("Introduction").